cli/installer.rs

```diff
@@ -101,17 +101,29 @@ fn get_installer_dir() -> Result<PathBuf, Error> {
 
 pub fn install(
   flags: DenoFlags,
-  installation_dir: Option<PathBuf>,
-  exec_name: &str,
-  module_url: &str,
+  exe_specifier: &str,
+  module: &str,
   args: Vec<String>,
   force: bool,
 ) -> Result<(), Error> {
-  let installation_dir = if let Some(dir) = installation_dir {
-    dir.canonicalize()?
-  } else {
-    get_installer_dir()?
+  let exe_path = PathBuf::from(exe_specifier);
+  let installation_dir = match exe_path.parent() {
+    Some(parent) => {
```

Comment:
Is it always the case that `PathBuf::from("foo").parent() == None` ?

Reply:
`PathBuf::from("foo").parent()` is `""`.
`PathBuf::from("/").parent()` is `None`, the error case further below.

https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.parent